backend/helpers/createToken.js

```diff
@@ -0,0 +1,14 @@
+const jwt = require('jsonwebtoken');
+const { SECRET_KEY } = require('../config');
+
+/** Return signed JWT from user data. */
+
+function createToken(user) {
+  const payload = {
+    user_id: user.id
```

Comment:
Need name in payload. If a biz, add restaurant name. If a user, add fullname. But payload has one generic name key.

Reply:
After discussing seems like we don't want personally identifiable information in the token, like the users actual name. Opting for account_type and user_id in the payload. 

Further discussion also prompted the idea to separate individuals into the users table, while businesses and charities will get their own tables in the database and corresponding models in the backend. PR soon.